lib/jmap/mail/calendar/reply/calendar_event_reply_response.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:jmap_dart_client/jmap/core/id.dart';
+import 'package:jmap_dart_client/jmap/core/method/response/calendar_event_reply_response.dart';
+import 'package:jmap_dart_client/jmap/mail/calendar/properties/event_id.dart';
+import 'package:jmap_dart_client/util/json_parsers.dart';
+
+class CalendarEventAcceptResponse extends CalendarEventReplyResponse {
+  CalendarEventAcceptResponse(
+    super.accountId,
+    super.notFound,
+    {
+      this.accepted,
+      this.notCreated
+    });
+
+  final List<EventId>? accepted;
+  final List<Id>? notCreated;
+
+  static CalendarEventAcceptResponse deserialize(Map<String, dynamic> json) {
+    return CalendarEventAcceptResponse(
+      JsonParsers().parsingAccountId(json),
+      JsonParsers().parsingListId(json, 'notFound'),
+      accepted: JsonParsers().parsingEventId(json, 'accepted'),
+      notCreated: JsonParsers().parsingListId(json, 'notCreated'),
+    );
+  }
+
+  @override
+  List<Object?> get props => [...super.props, accepted, notCreated];
+}
+
+class CalendarEventRejectResponse extends CalendarEventReplyResponse {
```

Comment:
We should separate it into 2 separate `dart files` for each class for easy maintenance later.

Reply:
`CalendarEventRejectResponse` shouldn't exist in this PR. Removed.

Reply:
Ok, please rename file name correctly

Reply:
Done